src/implementations/vanilla/components/expandable/_expandable.scss

```diff
@@ -16,11 +16,30 @@ $_expandable-content-color: null !default;
   transform: rotate(0deg);
 }
 
+.ecl-expandable__divider {
+  background-color: map.get(theme.$color, 'grey-25');
+  border: none;
+  display: none;
+  height: 1px;
+  margin: 0;
+}
+
 .ecl-expandable__content {
   color: $_expandable-content-color;
-  font-family: #{map.get(theme.$font-family, 'default')};
+  font: map.get(theme.$font-prolonged, 'm');
+  margin-top: map.get(theme.$spacing, 'm');
+  max-width: 80ch;
```

Comment:
I'm a little worried by this max width. It will work fine when there is a block of text in the expandable, but if there is something else (table, image, ...) it should probably take the full width

Reply:
Also, couldn't we set the max width on the child elements rather than on the container, and so use a margin-bottom instead of an additional hr?

Reply:
mmh..yep, although this is also valid for the left indentation, the truth is that on design side they always "deal" with text only inside this hidden container, it all originated from this..
So, i followed your suggestion, but it feels weak in any case, if we remove the max-width then we have to rely on a parapgraph tag wrapping the content, which is what has been done.